app/controllers/loan_requests_controller.rb

```diff
@@ -15,6 +17,25 @@ def create
     end
   end
 
+  def edit
+    @loan_request = LoanRequest.find(params[:id])
+  end
+
+  def show
+  end
+
+  def update
+    respond_to do |format|
+      if @loan_request.update(loan_request_params)
+        format.html { redirect_to loan_request_path(@loan_request), notice: 'Loan Request Updated' }
```

Comment:
Line is too long. [100/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.